opengrok-tools/src/main/python/opengrok_tools/mirror.py

```diff
@@ -34,6 +34,7 @@
 import os
 import sys
 import tempfile
+from multiprocessing import Pool, cpu_count
```

Comment:
is process pool better than thread pool?

Reply:
I think in this case it does not matter as calling `sync()` from `scm` modules will result in process spawning.